### PR TITLE
Bug fixes

### LINKED ISSUE #370: Bug in getSegment(va) (vivisect 1.0.0)
`vw.getSegment(0x4BA190)` for the file https://github.com/fireeye/capa-testfiles/blob/master/mimikatz.exe_ returns `None` with vivisect 1.0.0 while it used to return `(4939776, 16840, '.data', 'mimikatz')` in vivisect 0.1.0. As the `.data` section starts at 0x4B6000 and has a virtual size of 0x41C8, I think that the output of vivisect 0.1.0 was correct and that this is a bug introduced in the last release. As this makes capa failing and last time I tried it (I think with 38a3fc23dbe79cd55335b57aa3587c05f3fb1d72) that was not the case, I think this has been introduced in one of the last 5 commits.

## Steps to reproduce
```
import vivisect
file = 'mimikatz.exe_' # https://github.com/fireeye/capa-testfiles/blob/master/mimikatz.exe_
vw = vivisect.VivWorkspace()
vw.getSegment(0x4BA190)
```

### PR DESCRIPTION
**Pull request by rakuy0**
FIx #370 and #371 and some callgraph UI issues

### CODE REVIEW

**Review by atlas0fd00m — APPROVED** [review]
:+1:

File: setup.py [hunk 1]
```diff
@@ -32,11 +32,13 @@
...
-        # like have these as optional dependencies so we can install viv headless. Worth?
-        'pyqt5==5.15.1',
-        'pyqtwebengine==5.15.1',
     ],
+    extras_require={
+        'gui': [
+            'pyqt5==5.15.1',
+            'pyqtwebengine==5.15.1',
+        ]
+    },
```
Comment (williballenthin) [inline setup.py:41]: thank you!

**Comment by Ana06** [comment]
@rakuy0 are you planning to release v1.1.0 soon? I think this will fix https://github.com/fireeye/capa/issues/459, so a release would be appreciated 🥺

**Comment by rakuy0** [comment]
Aye. Soon. There's at least one last PR that fixes a few things for atlas that I promised I would hit. That should be going up for review soon, so with any luck getting eyes on it won't take too long and we can do a release relatively soon after that.

**Pull request merged**